src/SonarQube/SolutionBinding.sqconfig

```diff
@@ -1,4 +1,10 @@
 {
   "ServerUri": "https://dory.sonarsource.com/sonarqube/",
-  "ProjectKey": "sonarlint-visualstudio"
+  "ProjectKey": "sonarlint-visualstudio",
+  "Profiles": {
+    "CSharp": {
```

Comment:
We should not use SQ values (they could change). So should not be a pain if we'll have a stable display identifier which will have the same value.